internal/internal.go

```diff
@@ -400,6 +395,11 @@ func (p *envoyExtAuthzGrpcServer) check(ctx context.Context, req interface{}) (*
 		return nil, stop, err
 	}
 
+	if ctx.Err() != nil {
```

Comment:
We can move this before we convert to ast.Value correct ? We only need the input map to be non-nil.

Reply:
@ashutosh-narkar right, moved it right after parsing input.